files/default/neutron-ha-tool.py

```diff
@@ -43,14 +45,19 @@
 OS_PASSWORD_FILE = '/etc/neutron/os_password'
 
 
+IDENTITY_API_VERSIONS = {
+    '2.0': kclientv2,
+    '2': kclientv2,
+    '3': kclientv3
+}
+
+
 def parse_args():
     # ensure environment has necessary items to authenticate
-    for key in ['OS_TENANT_NAME', 'OS_USERNAME',
-                'OS_AUTH_URL', 'OS_REGION_NAME']:
+    for key in ['OS_USERNAME', 'OS_PASSWORD', 'OS_AUTH_URL',
+                'OS_ENDPOINT_TYPE']:
```

Comment:
Ah right. This comes from the backport. I tested it manually via command line, with the source .openrc and with the password file. Will recheck via RA.